hack/images/dcos/Makefile

```diff
@@ -2,7 +2,7 @@ FROM	?= /target
 TARGET_OBJ = km kubectl
 IMAGE	?= kubernetes
 REPO	?= mesosphere/$(IMAGE)
-TAG	?= k8s-0.14.2-k8sm-0.5-dcos-dev
+DCOS_VERSION ?= dev
 SUDO	?= $(shell test "$$(whoami)" = "root" || echo sudo)
```

Comment:
Instead of calling `sudo` maybe we could make a target that can check `whoami` and fail with a descriptive message.  We'd depend on that target from the targets that need root.  I'd rather have to explicitly type `sudo make ...`, but maybe that's just me.

Probably doesn't matter much if the plan is to eventually scrap this Makefile.

Guess it doesn't matter too much if this is